src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/wwwroot/index.html

```diff
@@ -6,6 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>ComponentsWebAssembly-CSharp</title>
     <base href="/" />
+    <link rel="preload" id="webassembly" />
```

Comment:
dotnet-wasm ? dotnet-webassembly ?

Reply:
I think changing the name would require changing the SDK side too. If we don't feel strongly about naming, I think going with webassembly is fine. (It's also more consistent with what we use elsewhere, where we just use webassembly if that helps)

Reply:
This name already landed in different PR for "hosted" mode.